GWAS/LD_Clumping.ipynb

```diff
@@ -302,10 +306,10 @@
     "[filter_samples: provides = f'{cwd}/{unrelated_samples:bn}.{ld_sample_size}.txt']\n",
     "input: unrelated_samples, sampleFile\n",
     "output: f'{cwd}/{unrelated_samples:bn}.{ld_sample_size}.txt'\n",
-    "R: expand = \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
+    "R: container = container_lmm, expand = \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
     "    set.seed(1)\n",
     "    all_unrelated = read.table(${_input[0]:r}, header=T)\n",
-    "    avail_samples = read.table(${_input[1]:r}, header=F, skip=2)\n",
```

Comment:
@dianacornejo why do we have `skip=2` previously? this change is clearly incompatible with previous code. Just want to double check on this.

Reply:
as per @gaow recommentation added parameter header
`avail_samples = read.table(${_input[1]:r}, header=F${", skip=2" if header else ""})`